types/react-select/lib/Creatable.d.ts

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import Select, { Props as SelectProps } from './Select';
+import { OptionType, OptionsType, ValueType, ActionMeta } from './types';
+import { cleanValue } from './utils';
+import manageState from './stateManager';
+
+export type CreatableProps = {
+  /* Allow options to be created while the `isLoading` prop is true. Useful to
+     prevent the "create new ..." option being displayed while async results are
+     still being loaded. */
+  allowCreateWhileLoading?: boolean,
+  /* Gets the label for the "create new ..." option in the menu. Is given the
+     current input value. */
+  formatCreateLabel?: (inputValue: string) => Node,
+  /* Determines whether the "create new ..." option should be displayed based on
+     the current input value, select value and options array. */
+  isValidNewOption?: (a: string, b: ValueType, c: OptionsType) => boolean,
```

Comment:
a is inputValue, b value, c options

Reply:
Thanks for all your comments, @Darep!

I'll update the signatures.